### DISCUSSION TITLE
Is the serializer still actively maintained? - laste release more than a year old...

### DISCUSSION THREAD

**Discussion by javafanboy** [activej/activej#321] [General]
I am looking for a serialization library and noted that the latest stable version of ActiveJ serializer on maven central is a year old and the beta2 6 is from Nov last year. A lot of things has happended in the Java world since then. Will there be a "stable 6" any time soon?

**Comment by eduard-vasinskyi** [comment]
Hi, @javafanboy

ActiveJ Serializer is actively used and maintained. The latest changes contained API improvements as well as support for Java Records.

As for the release of the stable ActiveJ 6 version - it is still being tested. We hope to release a stable version soon.
